engine/src/test/java/com/arcadedb/query/sql/executor/AlterTypeExecutionTest.java

```diff
@@ -102,4 +102,11 @@ public void sqlAlterTypeCustom() {
     Assertions.assertNull(database.getSchema().getType("Suv").getCustomValue("age"));
     Assertions.assertFalse(database.getSchema().getType("Suv").getCustomKeys().contains("age"));
   }
+
+  @Test
+  public void sqlAlterTypeName() {
+    database.command("sql", "CREATE VERTEX TYPE Mpv");
+    database.command("sql", "ALTER TYPE Mpv NAME Sedan");
+    Assertions.assertNotNull(database.getSchema().getType("Sedan"));
```

Comment:
It would be wonderful if you can also test the previous Mpv type doesn't exist anymore